partiql-tests-data/eval/primitives/operators/is-operators.ion

```diff
@@ -0,0 +1,122 @@
+is::[
+  {
+    name: "1.000 is DECIMAL(3,3)",
+    statement: "1.000 is DECIMAL(3,3)",
+    assert: {
+      result: EvaluationSuccess,
+      evalMode: [
+        EvalModeCoerce,
+        EvalModeError
+      ],
+      output: true
+    }
+  },
+  {
+    name: "1.0000 is DECIMAL(5,3)",
+    statement: "1.0000 is DECIMAL(5,3)",
+    assert: {
+      result: EvaluationSuccess,
+      evalMode: [
+        EvalModeCoerce,
+        EvalModeError
+      ],
+      output: false
+    }
+  },
+  {
+    name: "0.001 is DECIMAL(3,3)",
+    statement: "0.001 is DECIMAL(3,3)",
+    assert: {
+      result: EvaluationSuccess,
+      evalMode: [
+        EvalModeCoerce,
+        EvalModeError
+      ],
+      output: true
+    }
+  },
+  {
+    name: "123.456 is DECIMAL(2,2)",
+    statement: "123.456 is DECIMAL(2,2)",
```

Comment:
Could you please explain how `123.456` with three decimal places is considered a decimal with scale 2? Similarly, how is the precision only 2?